en/docs/guides/tenants/customize-automated-mails.md

```diff
@@ -117,6 +117,7 @@ Once you have set the template language for the user, it is also necessary to sp
     
         For example, use **fr\_fr** for the French locale, **pt\_br** for Brazil locale and **it\_it** for the Italian locale. This is the format used for country locales in the Identity Server.
         For more information on country locales, see [Locale codes](http://www.science.co.il/language/Locale-codes.php).
+        Please make sure to use the **"\_"** format instead of the **"-"** format used in this page.
```

Comment:
In that case, shall we use another link with the proper format? It will make the text less confusing and low chance of getting it wrong if users don't read this text.

Found this page: https://www.ibm.com/docs/en/radfws/9.7?topic=overview-locales-code-pages-supported

Reply:
I have updated the link with the above link as it would be much clearer for the user. But can you verify if it's okay to use a `www.ibm.com/docs` link in our official docs?